### ISSUE TITLE
Should warn when using unknown/strange attributes.

### ISSUE THREAD

**Issue by DrecDroid** [canjs/can-stache#139]
For example when using component bindings but have not imported `can-stache-bindings` It should warn about unrecognizable attributes during development.
e.g.

```javascript
var stache = require('can-stache');

var template = stache('<button ($click)="showMessage()">Click</button>');

document.body.appendChild(template({
    message: "Definetively won't be shown",
    showMessage:  function(){ alert(this.message);  }
  })
);
```
This will fail silently because `can-stache-binding` was not imported.

**Comment by justinbmeyer** [comment]
The place to do this is around here:

https://github.com/canjs/can-stache/blob/18bff6833232893d98cfc3d3bf2bfa7c8da178b2/can-stache.js#L234

Check if the attribute looks strange and if there's no `attrCallback`, provide a warning.

**Comment by Sinjhin** [comment]
This is fixed with PR #247
